airflow/dags/reporting_database_generation/generate-reporting-db.py

```diff
@@ -577,11 +582,23 @@ class SQLTemplatedPythonOperator(PythonOperator):
 #     params=database_name_config
 # )
 
+# OTHER MISC TASKS TO BE RUN
+create_granted_patent_crosswalk = PythonOperator(task_id='create_granted_patent_crosswalk',
```

Comment:
Are there no other DAGs that's is better suited for this task? Technically, this belong to "raw database" creation. Further the bulk download process is understood as dependent on raw database creation and considered ready to start as soon raw database is done. Having this task here may introduce inconsistencies